services/monkey/helpers.go

```diff
@@ -39,3 +43,79 @@ func (m *Monkey) storeLogs(r io.ReadSeeker) (string, error) {
 
 	return cid, nil
 }
+
+// Retry function with a custom condition
+func Retry(maxRetries int, waitBeforeRetry time.Duration, operation interface{}, condition interface{}, args ...interface{}) ([]interface{}, error) {
```

Comment:
tests are passing. however, i'm no sure using interface{} (should be `any` btw) is a good choice here.
make a retry just for our use case since it's used only once.

Reply:
Thanks for your comment. I have reverted and moved the retry logic to helpers. 
I just needed one advise on how to get the global variable for maxRetries and waitBeforeRetry? Or maybe I can do it in the next PR?